tools/wptrunner/wptrunner/formatters/chromium.py

```diff
@@ -28,11 +28,15 @@ def __init__(self):
         # the subtest messages for this test.
         self.messages = defaultdict(str)
 
+        # List of tests that have failing subtests.
+        self.tests_with_subtest_fails = []
```

Comment:
This should probably be a set instead of a list, so that all lookup/addition/removal will be O(1) instead of O(n). There is likely a noticeable difference in real-world performance because we have tests with tens of thousands of subtests.

```suggestion
        self.tests_with_subtest_fails = set()
```

Reply:
Good catch, done.